xtask/src/cli.rs

```diff
@@ -85,6 +87,10 @@ pub enum Action {
     WebCheck,
     WebInstall,
     WebRun,
+    FfiBuildDll {
+        debug: bool,
```

Comment:
nitpick: Use `release` since this is what we are interested into (instead of negating the value twice).
```suggestion
        release: bool,
```

Reply:
updated